CMakeLists.txt

```diff
@@ -67,6 +67,9 @@ add_definitions(-DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_${THRUST_DEVICE_SYST
 set(CMAKE_CXX_STANDARD 11 CACHE STRING "The C++ version to be used.")
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Ignore when building thrust itself on deprecated C++ standards
```

Comment:
Should we leave this here, or bump the CMAKE_CXX_STANDARD above to 14 and require CI to define this as needed?